styling-css-modules-sample/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "module": "CommonJS",
+    "target": "ES2017",
+    "lib": ["ESNext", "DOM", "DOM.Iterable"],
+    "moduleResolution": "Node",
+    "sourceMap": false,
+    "declaration": false,
+    "strict": false,
+    "noImplicitAny": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "experimentalDecorators": true,
+    "jsx": "react",
+    "plugins": [{ "name": "typescript-plugin-css-modules" }],
```

Comment:
Did't know about that field. How does it work? Where the list of available plugins?

Reply:
Here is the `plugin` field documentation https://www.typescriptlang.org/tsconfig#plugins
And here is `typescript-plugin-css-modules` page https://github.com/mrmckeb/typescript-plugin-css-modules